reactor-core/README.md

```diff
@@ -7,3 +7,4 @@ This module contains articles about Reactor Core.
 - [Intro To Reactor Core](https://www.baeldung.com/reactor-core)
 - [Combining Publishers in Project Reactor](https://www.baeldung.com/reactor-combine-streams)
 - [Programmatically Creating Sequences with Project Reactor](https://www.baeldung.com/flux-sequences-reactor)
+- [How To Get String From Mono In Reactive Java](http://baeldung.com/string-from-mono/)
```

Comment:
Please revert this change -- your editor will add the link after the article is published.